.github/scripts/build_daprbundle.py

```diff
@@ -109,6 +112,7 @@ def downloadBinary(repo, fileBase, version, out_dir):
 
     print(f"Downloaded {url} to {downloadPath}")
 
+# Downloads all required dapr binaries(`daprd`,`placement`,`dashboard`) in `BIN_DIR` subfolder and dapr cli(`dapr`) in `dir` folder
```

Comment:
nit: space before parenthesis start, comma after `e.g.`, same for other places.

Reply:
Done